tsconfig.json

```diff
@@ -31,9 +31,18 @@
       "~/*": [
         "src/*"
       ],
-    }
+    },
+    "plugins": [
+      {
+        "transform": "typescript-transform-paths"
+      },
+      {
+        "transform": "typescript-transform-paths",
```

Comment:
Thanks, this seems to do the trick. Can you help me understand why both transforms are required here? I tried a few variations with either/or and it looks like some of the resulting paths still include `~` unless both of these are present. It's not clear from the ttsc docs why both of these would be required.

Reply:
It's also a shame that tsc doesn't support transformations through `tsconfig.json`.

Reply:
The docs you are looking for are the [typescript-transform-paths](https://www.npmjs.com/package/typescript-transform-paths) docs. ttsc is just the "runner".

The first line transforms paths in `.js` files, the second line transforms paths in the `.d.ts` files.